packages/block-library/src/navigation/index.php

```diff
@@ -270,7 +270,7 @@ function block_core_navigation_get_classic_menu_fallback() {
 		}
 
 		// Returns a menu if it contains `primary` in its name/slug.
-		foreach ($classic_nav_menus as $classic_nav_menu) {
+		foreach ( $classic_nav_menus as $classic_nav_menu ) {
 			if ( str_contains( $classic_nav_menu->slug, 'primary' ) ) {
```

Comment:
Just checking - we definitely want to check for _any_ mention of `primary` rather than the specific string?

Reply:
```suggestion
			if ( 'primary' === $classic_nav_menu->slug ) {
```

Reply:
Thanks for the feedback, makes sense to me. I updated this in [2449f7a](https://github.com/WordPress/gutenberg/pull/45976/commits/2449f7a30975af230f894f44cead61a1fb43f77e) as well as the front-end behavior to match.

Reply:
Indeed, just the full match